generate_openocdcfg.py

```diff
@@ -123,7 +123,7 @@ def main(argv):
         parsed_args.dts, followIncludes=True)
 
     if "vcu118" in parsed_args.board:
-        adapter_khz = 1800
+        adapter_khz = 20000
     else:
         adapter_khz = 10000
```

Comment:
Any reason we should just use 20 across the board?

Reply:
(assuming you mean shouldn't)
I'm not sure that I've ever tested faster than 10 on arty, but we could if it's known to work

Reply:
Let it lie, especially since Arty is being "deprecated".